### PR TITLE
FI-742: add reference server to site-overlay

### PR DESCRIPTION
**Pull request by okeefm**
This PR adds the reference-server and its db to inferno-site-overlay. 

NOTE: To test this, you will have to pull down `inferno-reference-server` from this PR: https://github.com/inferno-community/inferno-reference-server/pull/4, build that branch with `docker build -t infernocommunity/inferno-reference-server:latest .`, prior to running `docker-compose up --force-recreate` in this branch.

ALSO: Be sure to use your local machine's IP, rather than `localhost`, during testing, so that the different applications can speak with one another.

﻿Pull requests into inferno-site-overlay require the following items to be completed. Submitter and reviewer 
should check the relevant check boxes when the associated item is done. For items that are not 
applicable, note it's not applicable ("N/A") and check the box. For example, external Pull 
Requests do not require ticket links.

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Internal ticket for this PR: https://oncprojectracking.healthit.gov/support/browse/FI-742
- [x] Internal ticket links to this PR
- [x] Internal ticket is properly labeled (Community/Program)
- [x] Internal ticket has a justification for its Community/Program label
- [x] Code diff has been reviewed for extraneous/missing code

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] You have tried to break the code

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -37,9 +37,9 @@ services:
     image: nginx
     volumes:
       - ./nginx/nginx.conf:/etc/nginx/nginx.conf:ro
-      - /etc/ssl/certs/inferno:/etc/ssl/certs/inferno:ro
+      # - /etc/ssl/certs/inferno:/etc/ssl/certs/inferno:ro
```
Comment (radamson) [inline docker-compose.yml]: It looks like changing this line is needed for testing locally (based on the comment below it), but we ultimately don't want to pull this into master right?

Comment (okeefm) [reply docker-compose.yml]: Yes, that's accurate. I meant to pull out all my testing changes, but missed this one. Reverting now.

Comment (radamson) [reply docker-compose.yml]: yup no worries 👍 .  Though I do still need to manually make this change to test locally with `docker build -t infernocommunity/inferno-reference-server:latest .` right?

**Comment by radamson** [comment]
I think it would be worth updating `inferno-config.yml` to include the server as a preset.  It looks like its already in there (although using localhost which needs to be changed) but I don't see the popup in the UI.

**Comment by okeefm** [comment]
Agreed. I'll add the reference server to inferno-config and community-config

**Review by radamson — APPROVED** [review]
Tested it out and everything works.

When deploying there's a couple things we should check

- That the database contains data and post the data to it if not
- make sure that the presets are working (which we can only reasonable do when deployed)

**Review by arscan — APPROVED** [review]

**Pull request merged**